app/src/test/java/com/kickstarter/viewmodels/AccountViewModelTest.kt

```diff
@@ -82,6 +82,18 @@ class AccountViewModelTest : KSRobolectricTestCase() {
         this.showEmailErrorIcon.assertValue(true)
     }
 
+    @Test
+    fun testShowEmailErrorIconGoneForBackerUndeliverable() {
```

Comment:
what's the difference between this test and `testShowEmailErrorIconForBackerUndeliverable`? This code is kind of hard to read because idk what these booleans correspond to `true, false, true`. So it's be helpful if we had variables like `val isCreator = false` for example and pass those in

Reply:
fixed!